README.md

```diff
@@ -1,5 +1,101 @@
-# Preferences Server
-General Preferences Service
+# Personal Data Server
 
-This project is for the code and documentation of a common general preferences
-service that can be used to store, retrieve, and modify user preferences.
+Personal Data Server can be used to save, retrieve, update and delete personal user data. This project is written in
+[`node.js`](https://nodejs.org/en/) and uses [`Postgres`](https://www.postgresql.org/) as the backend database.
+
+This project supports these authentication methods:
+
+* Google Single Sign-on
+
+## Getting Started
+
+To work on the project, you need to:
+
+* Install [NodeJS and NPM](https://nodejs.org/en/download/) for your operating system
+* Clone the project from GitHub. [Create a fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+with your GitHub account, then run the following in your command line (make sure to replace `your-username` with
+your username):
+
+```bash
+git clone https://github.com/your-username/preferencesServer
+```
+
+* Obtain Google OAuth 2.0 client credentials from the [Google API Console](https://console.developers.google.com/)
```

Comment:
Could there be a way of setting up a job in CI with a secret provided e.g. by an environment variable that would allow us to make this a kind of mostly automated test?